src/App.js

```diff
@@ -236,21 +230,6 @@ export default function App() {
                     )}
                   />
 
-                  {/**
-                   * This endpoint is to be able to track clicks to the COVID
-                   * Response Simulator on the tools page. The user will be briefly
-                   * redirected to COVID_RESPONSE_SIMULATOR_REDIRECT_URL and then
-                   * to the spreadsheet. The number of visits to the redirect URL
-                   * will correspond to the number of clicks to the COVID Response
-                   * Simulator.
-                   */}
-                  <Route
-                    path="/covid-response-simulator-redirect"
-                    component={() => (
-                      <ExternalRedirect url={COVID_RESPONSE_SIMULATOR_URL} />
-                    )}
-                  />
```

Comment:
Tested the absence of this `Route` element too. Using `covid-response-simulator-redirect` path would just redirect to the home page.